packages/lambda-tiler/src/routes/tile.style.json.ts

```diff
@@ -112,10 +115,20 @@ export async function styleJsonGet(req: LambdaHttpRequest<StyleGet>): Promise<La
     if (tileSet == null) return NotFound();
     if (tileSet.type !== TileSetType.Raster) return NotFound();
     return tileSetToStyle(req, tileSet, apiKey);
+  } else {
+    for (const layer of styleConfig.style.layers) {
+      if (excluded.has(layer.id.toLowerCase())) continue;
+      layers.push(layer);
+    }
   }
 
   // Prepare sources and add linz source
-  const style = convertStyleJson(styleConfig.style, apiKey, ConfigLoader.extract(req));
+  const style = convertStyleJson(
+    styleConfig.style,
+    apiKey,
+    ConfigLoader.extract(req),
+    layers.length > 0 ? layers : undefined,
```

Comment:
if you exclude everything, you get all the layers? but otherwise there will alwys be a layer array with everything in?

how about passing the filter into the function instead? then the return could just be `style.layers.filter(f => !excluded.has(f.id.toLowerCase()))`

Reply:
Just fixed.